cypress/integration/example_spec.js

```diff
@@ -253,6 +256,12 @@ describe('pipe()', () => {
             }
           })
       })
+
+      it('should use the body tag if no previous subject was provided', () => {
+        cy.pipe(getFirst)
```

Comment:
Previous subject wasn't defined. I verified the final type signature of `pipe` is required to prevent a type error on this line. The test ensures it passes at runtime.